Services/CO.CDP.EntityVerification.Tests/Events/EventDeserializerTest.cs

```diff
@@ -58,4 +69,28 @@ public void ItThrowsAnExceptionIfTheEventCannotBeDeserialized()
             ],
             Roles = ["supplier"]
         };
+
+    private static OrganisationUpdated GvienOrganisationUpdatedEvent() =>
+        new()
+        {
+            PponId = "2e1230b94ced4fa9aac76895f122e9f0",
```

Comment:
We discussed the `OrganisationUpdated` event should have same fields as `OrganisationRegistered` (both based on OCDS). We won't have a `PponId` in there. PPON is just one of the identifiers.

https://github.com/cabinetoffice/GCGS-Central-Digital-Platform/blob/5a4a7d77410d5eff281d28426605795c120ecbc9/Services/CO.CDP.Organisation.WebApi/Events/OrganisationUpdated.cs#L3-L20

Reply:
In the EntityVerification project you can just copy the `OrganisationRegistered` and rename it to `OrganisationUpdated`.

https://github.com/cabinetoffice/GCGS-Central-Digital-Platform/blob/5a4a7d77410d5eff281d28426605795c120ecbc9/Services/CO.CDP.EntityVerification/Events/InboundEvents.cs#L5-L20

Reply:
Done.